setup.py

```diff
@@ -0,0 +1,4 @@
+from setuptools import setup
+
+if __name__ == "__main__":
+    setup()
```

Comment:
Actually we do not need setupy.py, because of pyproject.toml file

Reply:
I originally thought I needed it for Nix but I found an override. I'll remove the commit